### PR TITLE
[EDU-4090] refactor: change l2 to tiered cache

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Modified L2 product name in all docs

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[EDU-4090]: https://aziontech.atlassian.net/browse/EDU-4090?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gabriel-alves-azion — APPROVED** [review]

File: src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx [hunk 1]
```diff
@@ -190,18 +190,18 @@ See each available field and their descriptions below.
...
+| cacheTtl | Time, in seconds, the cached object is considered valid (not expired). After the time expiration, when a new request occurs, Tiered Cache queries the data on the origin (upstream). Example: `31536000` |
 | configurationId | Unique Azion configuration identifier set on virtual host configuration file. Example: `1595368520` |
 | host | Host information sent on the request line. Stores: host name from the request line, or host name from the `Host` request header field, or the server name matching a request. Example: `g1sdetynmxe0ao.map.azionedge.net` |
 | proxyHost | Hostname being proxied. Example: `storage.googleapis.com:443` |
 | proxyStatus | HTTP error status code or origin when no response is obtained from the upstream. Example: `520`. **In case of cache, the response is `-`.** |
-| proxyUpstream | Origin (upstream) address. In some cases, the L2 origin can be Image Processor (IMS) to process the image and then cache it. Example: `ims_http` |
+| proxyUpstream | Origin (upstream) address. In some cases, the Tiered Cache origin can be Image Processor (IMS) to process the image and then cache it. Example: `ims_http` |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx:204]: vai esse (IMS) mesmo?

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx [hunk 2]
```diff
@@ -48,7 +48,7 @@ If your application isn't sending `Cache-Control` and `Expires` headers or the v
 4. Add the TTL in seconds in the **Maximum TTL (seconds)** field. For example: `864000` = 10 days.
 
 :::note
-The maximum and default TTL values for the edge should be between `60` and `31536000` seconds (1 year). If you have the [Application Acceleration](/en/documentation/products/build/edge-application/application-acceleration/) module activated, you can set this value to a minimum of `0`. Additionally, if you have the [L2 Caching](/en/documentation/products/edge-application/l2-caching/) module activated and wish to maintain cache in the L2, you must set TTL to at least `3` seconds.
+The maximum and default TTL values for the edge should be between `60` and `31536000` seconds (1 year). If you have the [Application Acceleration](/en/documentation/products/build/edge-application/application-acceleration/) module activated, you can set this value to a minimum of `0`. Additionally, if you have the [Tiered Cache](/en/documentation/products/edge-application/l2-caching/) module activated and wish to maintain cache in the Tiered Cache layer, you must set TTL to at least `3` seconds.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx:51]:
```suggestion
The maximum and default TTL values for the edge should be between `60` and `31536000` seconds (1 year). If you have the [Application Acceleration](/en/documentation/products/build/edge-application/application-acceleration/) module activated, you can set this value to a minimum of `0`. Additionally, if you have the [Tiered Cache](/en/documentation/products/edge-application/l2-caching/) module activated and wish to maintain cache in the tiered cache layer, you must set TTL to at least `3` seconds.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx:51]: por ser a layer e não o produto?

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx [hunk 3]
```diff
@@ -216,10 +216,10 @@ curl --location 'https://api.azionapi.net/edge_applications/<edge_application_id
   | `cache_by_cookies` | Defines whether cache should be segmented into [cache keys based on cookies](/en/documentation/products/edge-application/cache-settings/#cache-by-cookie). For all available options, see the [API documentation](https://api.azion.com). |
   | `cookie_names` | Sets the strings listed as the names of cookies that should be maintained as the same cache key or segmented into separate cache keys. |
   | `enable_caching_for_post` | Enables caching of `POST` requests. |
-  | `l2_caching_enabled` | Enables caching in the L2. You must have the [L2 Caching](/en/documentation/products/edge-application/l2-caching/) module active to enable this setting. |
+  | `l2_caching_enabled` | Enables caching in the Tiered Cache layer. You must have the [Tiered Cache](/en/documentation/products/edge-application/l2-caching/) module active to enable this setting. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx:219]:
```suggestion
  | `l2_caching_enabled` | Enables caching in the tiered cache layer. You must have the [Tiered Cache](/en/documentation/products/edge-application/l2-caching/) module active to enable this setting. |
```

File: src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx [hunk 4]
```diff
@@ -216,10 +216,10 @@ curl --location 'https://api.azionapi.net/edge_applications/<edge_application_id
...
   | `is_slice_configuration_enabled` | Enables the [file slicing](/en/documentation/products/edge-application/cache-settings/#slice-settings) feature. |
   | `is_slice_edge_caching_enabled` | Enables file slicing for the edge. If `is_slice_configuration_enabled` is set to `true`, this property must also be set to `true`. |
-  | `is_slice_l2_caching_enabled` | Enables file slicing for the L2. You must have the [L2 Caching](/en/documentation/products/edge-application/l2-caching/) module active to enable this setting. |
+  | `is_slice_l2_caching_enabled` | Enables file slicing for the Tiered Cache layer. You must have the [Tiered Cache](/en/documentation/products/edge-application/l2-caching/) module active to enable this setting. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/edit-edge-app/edit-cache-settings/cache-settings.mdx:222]:
```suggestion
  | `is_slice_l2_caching_enabled` | Enables file slicing for the tiered cache layer. You must have the [Tiered Cache](/en/documentation/products/edge-application/l2-caching/) module active to enable this setting. |
```

File: src/content/docs/en/pages/devtools/cli/azioncli-beta/azioncli-beta.mdx [hunk 5]
```diff
@@ -1029,7 +1029,7 @@ The `--slice-configuration-range` option gives the slice configuration range.
 
 ##### slice-l2-caching-enabled
 
-The `--slice-l2-caching-enabled` option informs whether slice L2 configuration is active or not.
+The `--slice-l2-caching-enabled` option informs whether slice for the Tiered Cache layer is active or not.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azioncli-beta/azioncli-beta.mdx:1032]:
```suggestion
The `--slice-l2-caching-enabled` option informs whether the slice for the tiered cache layer is active or not.
```

File: src/content/docs/en/pages/devtools/cli/azioncli-beta/azioncli-beta.mdx [hunk 6]
```diff
@@ -1131,7 +1131,7 @@ The `--slice-configuration-range` option gives the slice configuration range.
 
 ##### slice-l2-caching-enabled
 
-The `--slice-l2-caching-enabled` option informs whether slice L2 configuration is active or not.
+The `--slice-l2-caching-enabled` option informs whether slice for the Tiered Cache layer   is active or not.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azioncli-beta/azioncli-beta.mdx:1134]:
```suggestion
The `--slice-l2-caching-enabled` option informs whether the slice for the tiered cache layer is active or not.
```

File: src/content/docs/en/pages/devtools/cli/azioncli-beta/cache-settings-command/cache-settings-command.mdx [hunk 7]
```diff
@@ -153,7 +153,7 @@ The `--slice-configuration-range` option gives the slice configuration range.
 
 #### slice-l2-caching-enabled
 
-The `--slice-l2-caching-enabled` option informs whether slice L2 configuration is active or not.
+The `--slice-l2-caching-enabled` option informs whether slice for the Tiered Cache layer is active or not.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azioncli-beta/cache-settings-command/cache-settings-command.mdx:156]:
```suggestion
The `--slice-l2-caching-enabled` option informs whether the slice for the tiered cache layer is active or not.
```

File: src/content/docs/en/pages/devtools/cli/azioncli-beta/cache-settings-command/cache-settings-command.mdx [hunk 8]
```diff
@@ -255,7 +255,7 @@ The `--slice-configuration-range` option gives the slice configuration range.
...
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/cli/azioncli-beta/cache-settings-command/cache-settings-command.mdx:258]:
```suggestion
The `--slice-l2-caching-enabled` option informs whether the slice for the tiered cache layer is active or not.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/cache-settings.mdx [hunk 9]
```diff
@@ -49,9 +49,9 @@ With **Cache Settings** for **Edge Application**, you can edit cache configurati
 When activating the Slice functionality, the current *cache key* will be ignored and a new one will be created.
 :::
 
-Slice can be applied to both the edge caching layer or the L2, provided that **L2 Caching** is an active module.
+Slice can be applied to both the edge caching layer or the Tiered Cache layer, provided that **Tiered Cache** is an active module.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/cache-settings.mdx:52]:
```suggestion
Slice can be applied to both the edge caching layer and the tiered cache layer, provided that **Tiered Cache** is an active module.
```

File: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx [hunk 10]
```diff
@@ -221,19 +221,19 @@ import EdgeCaching from "~/includes/docs_help_center/en/real-time-metrics/edge-a
 
 ![Edge Caching graph information flow for Edge Applications, representing all data being transferred from both Edge Application In and Edge Application Out.](/assets/docs/images/uploads/edge-applications.png)
 
-If you have [L2 Caching](/en/documentation/products/build/edge-application/edge-caching/#l2-caching) enabled, your Edge Caching graph will exhibit:
+If you have [Tiered Cache](/en/documentation/products/build/edge-application/l2-caching/) enabled, your Edge Caching graph will exhibit:
 
-- **Edge Applications In**: data transferred from the end user to the edges, and from the edges to L2 cache.
+- **Edge Applications In**: data transferred from the end user to the edges, and from the edges to the Tiered Cache layer.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx:226]:
```suggestion
- **Edge Applications In**: data transferred from the end user to the edges, and from the edges to the tiered cache layer.
```

File: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx [hunk 11]
```diff
@@ -221,19 +221,19 @@ import EdgeCaching from "~/includes/docs_help_center/en/real-time-metrics/edge-a
...
 
-![Edge Caching with L2 enabled graph information flow for Edge Application In, representing data being transferred from the end user to the edges and from the edges to the L2 cache.](/assets/docs/images/uploads/l2-enabled-edge-applications-in.png)
+![Edge Caching with Tiered Cache enabled graph information flow for Edge Application In, representing data being transferred from the end user to the edges and from the edges to the Tiered Cache layer.](/assets/docs/images/uploads/l2-enabled-edge-applications-in.png)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx:228]:
```suggestion
![Edge Caching with Tiered Cache enabled graph information flow for Edge Application In, representing data being transferred from the end user to the edges and from the edges to the tiered cache layer.](/assets/docs/images/uploads/l2-enabled-edge-applications-in.png)
```

File: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx [hunk 12]
```diff
@@ -221,19 +221,19 @@ import EdgeCaching from "~/includes/docs_help_center/en/real-time-metrics/edge-a
...
 
-- **Edge Applications Out**: data transferred from L2 cache to the edges, and from the edges to the end user.
+- **Edge Applications Out**: data transferred from the Tiered Cache layer to the edges, and from the edges to the end user.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx:230]:
```suggestion
- **Edge Applications Out**: data transferred from the tiered cache layer to the edges, and from the edges to the end user.
```

File: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx [hunk 13]
```diff
@@ -221,19 +221,19 @@ import EdgeCaching from "~/includes/docs_help_center/en/real-time-metrics/edge-a
...
 
-![Edge Caching with L2 enabled graph information flow for Edge Application Out, representing data being transferred from the L2 cache to the edges and from the edges to the end user.](/assets/docs/images/uploads/l2-enabled-edge-applications-out.png)
+![Edge Caching with Tiered Cache enabled graph information flow for Edge Application Out, representing data being transferred from the Tiered Cache layer to the edges and from the edges to the end user.](/assets/docs/images/uploads/l2-enabled-edge-applications-out.png)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx:232]:
```suggestion
![Edge Caching with Tiered Cache enabled graph information flow for Edge Application Out, representing data being transferred from the tiered cache layer to the edges and from the edges to the end user.](/assets/docs/images/uploads/l2-enabled-edge-applications-out.png)
```

File: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx [hunk 14]
```diff
@@ -314,27 +314,27 @@ import BandwidthSaving from "~/includes/docs_help_center/en/real-time-metrics/ed
...
+**Tiered Cache** flow:
 
-![L2 Caching graph information flow for L2 Caching, representing all data being transferred from both L2 Caching In and L2 Caching Out.](/assets/docs/images/uploads/l2-caching.png)
+![Tiered Cache graph information flow for Tiered Cache, representing all data being transferred from both Tiered Cache In and Tiered Cache Out.](/assets/docs/images/uploads/l2-caching.png)
 
-**L2 Caching In** flow:
+**Tiered Cache In** flow:
 
-![L2 Caching graph information flow for L2 Caching In, representing data being transferred from the edges to L2 cache and from L2 cache to the client’s origin.](/assets/docs/images/uploads/l2-caching-in.png)
+![Tiered Cache graph information flow for Tiered Cache In, representing data being transferred from the edges to the Tiered Cache layer and from Tiered Cache to the client’s origin.](/assets/docs/images/uploads/l2-caching-in.png)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx:333]:
```suggestion
![Tiered Cache graph information flow for Tiered Cache In, representing data being transferred from the edges to the tiered cache layer and from Tiered Cache to the client’s origin.](/assets/docs/images/uploads/l2-caching-in.png)
```

File: src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx [hunk 15]
```diff
@@ -314,27 +314,27 @@ import BandwidthSaving from "~/includes/docs_help_center/en/real-time-metrics/ed
...
 
-**L2 Caching Out** flow:
+**Tiered Cache Out** flow:
 
-![L2 Caching graph information flow for L2 Caching Out, representing data being transferred from the client’s origin to the L2 cache and from the L2 cache to the edges.](/assets/docs/images/uploads/l2-caching-out.png)
+![Tiered Cache graph information flow for Tiered Cache Out, representing data being transferred from the client’s origin to the Tiered Cache layer and from the Tiered Cache to the edges.](/assets/docs/images/uploads/l2-caching-out.png)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/observe/real-time-metrics/real-time-metrics.mdx:337]:
```suggestion
![Tiered Cache graph information flow for Tiered Cache Out, representing data being transferred from the client’s origin to the tiered cache layer and from the Tiered Cache to the edges.](/assets/docs/images/uploads/l2-caching-out.png)
```

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/cache-settings.mdx [hunk 16]
```diff
@@ -48,7 +48,7 @@ Se a sua aplicação não estiver enviando cabeçalhos `Cache-Control` e `Expire
 4. Adicione o TTL em segundos no campo **Maximum TTL (seconds)**. Por exemplo: `864000` = 10 dias.
 
 :::note
-Os valores máximo e padrão de TTL para o edge devem estar entre `60` e `31536000` segundos (1 ano). Se você tiver o módulo [Application Acceleration](/pt-br/documentacao/produtos/edge-application/application-acceleration/) ativado, você pode definir este valor para um mínimo de `0`. Além disso, se você tiver o módulo [L2 Caching](/pt-br/documentacao/produtos/edge-application/l2-caching/) ativado e desejar manter o cache no L2, você deve definir o TTL para pelo menos `3` segundos.
+Os valores máximo e padrão de TTL para o edge devem estar entre `60` e `31536000` segundos (1 ano). Se você tiver o módulo [Application Acceleration](/pt-br/documentacao/produtos/edge-application/application-acceleration/) ativado, você pode definir este valor para um mínimo de `0`. Além disso, se você tiver o módulo [Tiered Cache](/pt-br/documentacao/produtos/edge-application/l2-caching/) ativado e desejar manter o cache na camada Tiered Cache, você deve definir o TTL para pelo menos `3` segundos.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/edite-edge-app/cache-settings.mdx:51]:
```suggestion
Os valores máximo e padrão de TTL para o edge devem estar entre `60` e `31536000` segundos (1 ano). Se você tiver o módulo [Application Acceleration](/pt-br/documentacao/produtos/edge-application/application-acceleration/) ativado, você pode definir este valor para um mínimo de `0`. Além disso, se você tiver o módulo [Tiered Cache](/pt-br/documentacao/produtos/edge-application/l2-caching/) ativado e desejar manter o cache na camada de Tiered Cache, você deve definir o TTL para pelo menos `3` segundos.
```

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/cache-settings.mdx [hunk 17]
```diff
@@ -58,7 +58,7 @@ TTL zero não é o mesmo que cache bypass. Para mais detalhes, leia sobre a dife
 Em vez de baixar um arquivo de conteúdo grande e correr o risco de timeouts ou terminações de conexão, [os arquivos podem ser fatiados](/pt-br/documentacao/produtos/edge-application/cache-settings/#slice-settings) em pedaços e armazenados em cache sob demanda.
 
 5. Em **Slice Settings**, habilite **Slice Configuration**.
-6. A camada **Edge Caching** já está selecionada por padrão. Se você tiver o módulo **L2 Caching** ativado, você também pode habilitar o slice para a L2.
+6. A camada **Edge Caching** já está selecionada por padrão. Se você tiver o módulo **Tiered Cache** ativado, você também pode habilitar o slice para a camada Tiered Cache.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/edite-edge-app/cache-settings.mdx:61]:
```suggestion
6. A camada **Edge Caching** já está selecionada por padrão. Se você tiver o módulo **Tiered Cache** ativado, você também pode habilitar o slice para a camada de Tiered Cache.
```

File: src/content/docs/pt-br/pages/build-jornada/edite-edge-app/cache-settings.mdx (hunk 16 above)
Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/build-jornada/edite-edge-app/cache-settings.mdx:51]: se concordar, revisar em PT

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**